tgstation.dme

```diff
@@ -7211,11 +7211,6 @@
 #include "modular_skyrat\modules\black_mesa\code\mobs\mob_spawners.dm"
 #include "modular_skyrat\modules\black_mesa\code\mobs\nihilanth.dm"
 #include "modular_skyrat\modules\black_mesa\code\mobs\vortigaunt.dm"
-#include "modular_skyrat\modules\blastwave_outfits\code\cargo_packs.dm"
-#include "modular_skyrat\modules\blastwave_outfits\code\clothing\blastwave_head.dm"
-#include "modular_skyrat\modules\blastwave_outfits\code\clothing\blastwave_mask.dm"
-#include "modular_skyrat\modules\blastwave_outfits\code\clothing\blastwave_suit.dm"
-#include "modular_skyrat\modules\blastwave_outfits\code\clothing\blastwave_uniform.dm"
```

Comment:
Please don't untick existing files in the codebase. Try and see if the code in them can be used, otherwise keep them ticked to avoid issues but just comment out all the code inside them